tests/suite/styling/in-block.typ

```diff
@@ -0,0 +1,12 @@
+--- warn-show-set-last-in-block ---
+#{
+// Warning: 1-14 show rule has no effect
+// Hint: 1-14 See https://typst.app/docs/tutorial/making-a-template/#set-and-show-rules
```

Comment:
I think it would be nicer to have an inline explanation, e.g.

`a show rule is only in effect until the end of the surrounding code block`

could you also indent the content of the code blocks by 2 spaces?